src/pretix/plugins/badges/exporters.py

```diff
@@ -225,10 +227,43 @@ def render_pdf(event, positions, opt):
     # place n-up badges/pages per page
     badges_pdf = PdfReader(outbuffer)
     nup_pdf = PdfWriter()
+    nup_pdf.add_metadata({
+        '/Title': 'Badges',
+        '/Creator': 'pretix',
+    })
     nup_page = None
+    max_nup_pages = 100
```

Comment:
I'd actually rather err on the lower end, maybe 20. If the customer uploads a background layout with images, a page can be already quite large. Or do you see a big downside to that?

Reply:
Yeah, you are right, for large backgrounds this gets big very quickly. I thought about how to better handle these cases instead of a static number, maybe call `sys.getsizeof(nup_pdf)` and split on a certain size (but which?). But probably 20 pages is enough as well. I just did not want to shift too much from memory-heavy to disk-io-heavy.